packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.controller.js

```diff
@@ -462,6 +475,87 @@ export default class OverTheBoxDetailsCtrl {
       });
   }
 
+  /**
+   * Get last seen values
+   */
+  getLastSeen() {
+    if (this.device && this.device.lastSeen) {
```

Comment:
This test should be done outside of the method. (You can even merge the first test with one at L462)